### PR TITLE
[Sprintr-001]  Login Page Initial Design

### PR DESCRIPTION
**Pull request by andela9**

### CODE REVIEW

File: frontend/app/styles/login.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import {greenA700, teal900} from "material-ui/styles/colors";
+
+// console.log(colors)
```
Comment (xolram) [inline frontend/app/styles/login.js]: Remove commented line

File: frontend/app/components/css/style.js [hunk 2]
```diff
@@ -1,28 +0,0 @@
...
-        margin: 30
-    },
-    h1: {
-        textAlign: 'left',
-        marginLeft: '30'
-
-    }
-};
-
-module.exports = style;
```
Comment (xolram) [inline frontend/app/components/css/style.js:28]: Add new line at the end

**Review by xolram — APPROVED** [review]

File: frontend/app/styles/login.js (hunk 1 above)
Comment (roomgee) [inline frontend/app/styles/login.js]: is this still needed?

File: frontend/app/components/Login.jsx [hunk 3]
```diff
@@ -50,16 +50,38 @@ export default class Login extends React.Component {
...
+                                        floatingLabelFocusStyle={styles.floatingLabelStyle}/>
+                                    <RaisedButton label="Log In" 
+                                        primary={true} 
+                                        labelStyle={styles["loginButtonLabel"]}                                        
+                                        style={styles["loginButtonOverlay"]} 
+                                        overlayStyle={styles["loginButton"]} 
+                                        buttonStyle={styles["loginButton"]} 
+                                        onClick={this.onClick}/>
+                                </form>
+
```
Comment (roomgee) [inline frontend/app/components/Login.jsx]: extra newline

**Review by roomgee — APPROVED** [review]

File: frontend/app/components/css/style.js (hunk 2 above)
Comment (andela9) [reply frontend/app/components/css/style.js:28]: this file was deleted. i applied this comment applicable files.

**Pull request merged**